docs/development/inlong_manager_shiro_plugin.md

```diff
@@ -0,0 +1,50 @@
+---
+title: InlongManager Shiro plugin development
```

Comment:
Custom Authentication

Reply:
ok,i will change it to "Custom Authentication"